data/sql_updates/create_committee_history.sql

```diff
@@ -21,9 +21,9 @@ with
         group by committee_id
     ),
     dcp_original as (
```

Comment:
Since this isn't pulling from `dimcmteproperties`, let's rename this alias to something that's more descriptive, like `filings` or `filings_original`.